src/instructlab/eval/mmlu.py

```diff
@@ -27,7 +91,7 @@ class MMLUEvaluator(Evaluator):
     def __init__(
         self,
         model_path,
-        tasks: list[str],
+        tasks: tuple[str, ...] = MMLU_TASKS,
```

Comment:
is that the type-annotation for "a tuple of one or more strings"? I haven't seen that before

Reply:
Yessir it is